prime-dotnet-webapi/HttpClients/Keycloak Administration/KeycloakAdministrationClient.cs

```diff
@@ -100,7 +103,18 @@ public async Task<bool> UpdateUserInfo(Guid userId, string email, string phoneNu
                 return false;
             }
 
-            return true;
+            bool success = true;
+            if (party.PartyTypes.Contains(PartyType.Labtech))
+            {
+                success &= await AssignRealmRole(userId, Roles.PhsaLabtech);
+            }
+
+            if (party.PartyTypes.Contains(PartyType.Immunizer))
+            {
+                success &= await AssignRealmRole(userId, Roles.PhsaImmunizer);
+            }
+
+            return success;
```

Comment:
It seems public method UpdatePhsaUserInfo will only assign labtech and immunizer roles which is very business specific. Maybe a method name with specific operation suffix is better since the generic name might cause confusion that it will update using the all info of input param "party". This is totally optional since this method is only used by phsa module.

Reply:
If this was UpdateUserInfo I'd agree. It's very specific to the eforms registration and may be refactored in the future if we do other provisioning for other systems. imo no changes required.